.github/workflows/java-cd.yml

```diff
@@ -196,8 +206,16 @@ jobs:
             - name: Move files to the correct directory tree
               run: |
                   mkdir -p build/io/valkey/valkey-glide/${{ env.RELEASE_VERSION }}
-                  cp -a maven-files/* build/io/valkey/valkey-glide/${{ env.RELEASE_VERSION }}
+                  mkdir -p build/io/valkey/valkey-glide-jedis-compatibility/${{ env.RELEASE_VERSION }}
+
+                  # Move client files
+                  cp -a maven-files/valkey-glide* build/io/valkey/valkey-glide/${{ env.RELEASE_VERSION }}
+
+                  # Move jedis-compatibility files
+                  cp -a maven-files/valkey-glide-jedis-compatibility* build/io/valkey/valkey-glide-jedis-compatibility/${{ env.RELEASE_VERSION }}
+
                   rm -rf build/io/valkey/valkey-glide/${{ env.RELEASE_VERSION }}/META-INF
+                  rm -rf build/io/valkey/valkey-glide-jedis-compatibility/${{ env.RELEASE_VERSION }}/META-INF
                   cd build
                   zip -r ../build io
```

Comment:
when publishing, both jars will be in the same bundle build.zip. Is that what we want? or we want separate this in two bundles?
check `- name: Publish to Maven Central` step

@jduo, can you take a look here. I don't recall the exact behaviour when publishing and using the dependencies from the maven repo.

@prateek-kumar-improving, Can you create a example project as well?